torch/quantization/fx/quantize.py

```diff
@@ -298,7 +304,7 @@ def get_qconfig(module_name):
                 self.modules[node.target].qconfig = module_qconfig
                 self.qconfig_map[node.name] = module_qconfig
 
-    def _prepare(self, model, qconfig_dict, inplace, is_dynamic_quant):
+    def _prepare(self, model, qconfig_dict, inplace, is_dynamic_quant, is_standalone_module):
```

Comment:
I understand that this is duplicating documentation, but can we add a quick comment here with what standalone module means?  It would improve readability and remove the need to navigate to a separate file to understand this argument.